### ISSUE TITLE
There is no total score in modsec_audit.log

### ISSUE THREAD

**Issue by trackscorer** [SpiderLabs/ModSecurity#2329]
**Describe the bug**
I add the rule in the modsecurity.conf:
```
SecAction "id:980145,phase:5,pass,t:none,log,noauditlog,\
    msg:'Incoming and Outgoing Score: %{TX.ANOMALY_SCORE} %{TX.OUTBOUND_ANOMALY_SCORE}'"
```
but now there is no "Incoming and Outgoing Score" in modsec_audit.log.

Before——modsecurity version v3,modsecurit-nginx v1.0.0:
```
ModSecurity: Warning. Matched "Operator `PmFromFile' with parameter `scanners-user-agents.data' against variable `REQUEST_HEADERS:User-Agent' (Value: `Mozilla/5.0 zgrab/0.x' ) [file "/etc/nginx/crs/rules/REQUEST-913-SCANNER-DETECTION.conf"] [line "33"] [id "913100"] [rev ""] [msg "Found User-Agent associated with security scanner"] [data "Matched Data: zgrab found within REQUEST_HEADERS:User-Agent: mozilla/5.0 zgrab/0.x"] [severity "2"] [ver "OWASP_CRS/3.2.0"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-reputation-scanner"] [tag "OWASP_CRS"] [tag "OWASP_CRS/AUTOMATION/SECURITY_SCANNER"] [tag "WASCTC/WASC-21"] [tag "OWASP_TOP_10/A7"] [tag "PCI/6.5.10"] [hostname "x.x.x.x"] [uri "/"] [unique_id "158928218340.100580"] [ref "o12,5v122,21t:lowercase"]
ModSecurity: Warning.  [file "/etc/nginx/modsec/modsecurity.conf"] [line "85"] [id "980145"] [rev ""] [msg "Incoming and Outgoing Score: 5 0"] [data ""] [severity "0"] [ver ""] [maturity "0"] [accuracy "0"] [hostname "x.x.x.x"] [uri "/"] [unique_id "158928218340.100580"] [ref ""]
```

Now——modsecurity version v3.0.4,modsecurit-nginx v1.0.1:
```
ModSecurity: Warning. Matched "Operator `PmFromFile' with parameter `scanners-user-agents.data' against variable `REQUEST_HEADERS:User-Agent' (Value: `Mozilla/5.0 zgrab/0.x' ) [file "/etc/nginx/crs/rules/REQUEST-913-SCANNER-DETECTION.conf"] [line "33"] [id "913100"] [rev ""] [msg "Found User-Agent associated with security scanner"] [data "Matched Data: zgrab found within REQUEST_HEADERS:User-Agent: mozilla/5.0 zgrab/0.x"] [severity "2"] [ver "OWASP_CRS/3.2.0"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-reputation-scanner"] [tag "OWASP_CRS"] [tag "OWASP_CRS/AUTOMATION/SECURITY_SCANNER"] [tag "WASCTC/WASC-21"] [tag "OWASP_TOP_10/A7"] [tag "PCI/6.5.10"] [hostname "x.x.x.x"] [uri "/"] [unique_id "159065854376.799948"] [ref "o12,5v82,21t:lowercase"]
```

**Comment by martinhsv** [comment]
Hi @trackscorer ,

The rule that you have added (980145) explicitly includes 'noauditlog' as one of its actions.

There were some bugs related to audit log output that were corrected in v3.0.4 that could explain why you were seeing incorrectly the message in the audit log prior to v3.0.4.

**Comment by trackscorer** [comment]
> Hi @trackscorer ,
> 
> The rule that you have added (980145) explicitly includes 'noauditlog' as one of its actions.
> 
> There were some bugs related to audit log output that were corrected in v3.0.4 that could explain why you were seeing incorrectly the message in the audit log prior to v3.0.4.

Cool, thank you.
I changed it to 'auditlog'.
